code/index.html

```diff
@@ -2,21 +2,122 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Uplifting News!</title>
+    <link rel="icon" href="images/book.png" />
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link
+      href="https://fonts.googleapis.com/css?family=Major+Mono+Display&display=swap"
+      rel="stylesheet"
+    />
+    <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
   </head>
+  <!-- Icons made by <a href="https://www.flaticon.com/authors/flat-icons" title="Flat Icons">Flat Icons</a> from <a
+  href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com</a> -->
+
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
-    </header>
 
-    <section class="big-news">
+      <div class="left-header-container">
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+
+        <img src="images/book.png" alt="Newspaper logo" />
+      </div>
+
+      <div class="container-upliftingnews">
+        <h1>Uplifting News</h1>
+      </div>
+
+      <nav>
+        <a href="">World</a>
+        <a href="">Local</a>
+        <h1>Uplifting News</h1>
+        <a href="">Animals</a>
+        <a href="">Subscribe</a>
+      </nav>
+    </header>
+    <div class="center-news-containers">
       <!-- The big news content -->
-    </section>
+      <section class="big-news-container">
+        <h2>
+          Research shows: Getting deliveries from UPS dogs instead of humans
+          will increase your life span with 2000%.
+        </h2>
+        <div>
+          <img src="images/ups-doggo.jpg" alt="Headline picture" />
+          <p>
+            Thunderbolts lilith x-force moonknight. Chimera northstar jameson
+            callisto typhoidmary ink thing moltenman, namorita zola. Magneto
+            crossbones ironfist patriot firestar meltdown johnnystorm?
+            Nightcrawler lukecage mrfantastic howard wildchild, whiplash
+            quicksilver frankcastle darkstar tchalla foggy sentinels. Mimic tusk
+            vanisher mockingbird shang-chi riptide ultron lionheart thunderbolts
+            havok rasputin aim kronos. Shocker nightcrawler arachne blackwidow
+            mistyknight nickfury cyber strange. Strange gambit sif yellowjacket.
+            X-force piledriver blade paladin warlock. Korath frankenstein ff
+            shiva alpha-flight sabretooth bluemarvel redhulk toxin pandemic
+            skreet nefaria proteus. Redskull scrambler manwolf namorita hydra
+            warbird ezekiel silversurfer magus. She-hulk deadpool patriot.
+          </p>
+        </div>
+      </section>
+      <!-- Other news  -->
+      <div class="small-news-container">
+        <section class="small-news-card">
+          <div>
+            <h3>These are tiny news!</h3>
+            <img src="images/puppy.jpg" alt="Small news image" />
```

Comment:
The reason why we have `alt` as props on images is to describe what the image is on for those that can't see the image. For example "puppy" is a good description when the image is on a puppy